CONTRIBUTING.md

```diff
@@ -301,7 +301,7 @@ Stacked PRs (Y -> X -> `develop`) must never be merged until their base is final
 After merging the parent PR, child's base changes automatically (see [GitHub blogpost](https://github.blog/changelog/2020-05-19-pull-request-retargeting/))
 
 To create a release:
-* make sure [CHANGE-NOTES.html](CHANGE-NOTES.html) are updated
+* make sure [CHANGE-NOTES.md](CHANGE-NOTES.md) are updated with `./gradlew patchChangeLog`
```

Comment:
Actually... what does this task specifically do? Just replace `## [Unreleased]` with the current version?